packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -421,7 +434,8 @@ export const Webchat = forwardRef((props, ref) => {
     if (!CoverComponent) return
     if (
       !botonicState ||
-      (botonicState.messages && botonicState.messages.length == 0)
+      (JSON.parse(botonicState).messages &&
+        JSON.parse(botonicState).messages.length == 0)
```

Comment:
Is `JSON.parse` really needed? I had a quick review of the library you're using here (https://github.com/soyguijarro/react-storage-hooks) and it seems it already parses the result from the storage automatically.

If you really need to parse it manually, please do it once right after the botonicState is loaded to avoid repeating a potential costly operation.